src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseViewType.java

```diff
@@ -51,10 +53,15 @@ public void setJurisdiction(CaseViewJurisdiction jurisdiction) {
         this.jurisdiction = jurisdiction;
     }
 
+    public String getPrintableDocumentUrl() { return printableDocumentUrl; }
+
+    public void setPrintableDocumentUrl(String printableDocumentUrl) { this.printableDocumentUrl = printableDocumentUrl; }
```

Comment:
I would normally expect the indentation to be preserved, but if it is accepted as per the team's / reform's standard, than it's ok to me.

Reply:
Agreed, but since it is maintained this standard in the rest of the places, I followed the same.